app/pages/project/classify.cjsx

```diff
@@ -1,318 +0,0 @@
-auth = require 'panoptes-client/lib/auth'
-React = require 'react'
-PropTypes = require 'prop-types'
-createReactClass = require 'create-react-class'
-ReactDOM = require 'react-dom'
-{ Helmet } = require 'react-helmet'
-apiClient = require 'panoptes-client/lib/api-client'
-counterpart = require 'counterpart'
-`import FinishedBanner from './finished-banner';`
-Classifier = require('../../classifier').default
-seenThisSession = require '../../lib/seen-this-session'
-ClassificationQueue = require('../../lib/classification-queue').default
-`import WorkflowAssignmentDialog from '../../components/workflow-assignment-dialog';`
-{ Split } = require('seven-ten')
-
-counterpart.registerTranslations 'en',
-  classifyPage:
-    title: 'Classify'
-
-# Map each project ID to a promise of its last randomly-selected workflow ID.
-# This is to maintain the same random workflow for each project when none is specified by the user.
-currentWorkflowForProject = {}
-
-# Map a workflow ID to a promise of its current classification resource
-# This is to maintain the same classification for each workflow.
-# In the future user might be able to specify subject sets, which we'll record here similarly.
-currentClassifications =
-  forWorkflow: {}
-
-# Queue up subjects to classify here.
-upcomingSubjects =
-  forWorkflow: {}
-
-emptySubjectQueue = ->
-  console?.log 'Emptying upcoming subjects queue'
```

Comment:
A lot (but not all) `console` statements in the original had the question mark operator with it. I changed them to just `console.log` in JS. When and why would this not be present?

Reply:
If you're not running the script in the browser. It was left over from when Brian published the entirety(!) of PFE as a npm package and it used to be imported into node scripts like the old panoptes-subject-uploader. Checking for it still could be useful if we're ever in the place to reuse code with the native app, but until we're at that place, I wouldn't worry about it.